dashboard/app/views/levels/editors/_netsim.html.haml

```diff
@@ -0,0 +1,67 @@
+%h2 NetSim settings
+
+%h3 Lobby configuration
+.field
+  %table.checkboxes
+    %tr
+      %td= boolean_check_box f, :show_clients_in_lobby
+      %td= f.label :show_clients_in_lobby, 'Show Clients in Lobby'
+      %td Whether client nodes should appear in the lobby list at all.
+    %tr
+      %td= boolean_check_box f, :show_routers_in_lobby
+      %td= f.label :show_routers_in_lobby, 'Show Routers in Lobby'
+      %td Whether router nodes should appear in the lobby list at all.
+    %tr
+      %td= boolean_check_box f, :show_add_router_button
+      %td= f.label :show_add_router_button, 'Show "Add Router" Button'
+      %td Whether the "Add Router" button should be available to students
+
+%h3 Send a Message Panel
+.field
+  %table.checkboxes
+    %tr
+      %td= boolean_check_box f, :show_packet_size_control
+      %td= f.label :show_packet_size_control, 'Show packet size control'
+      %td Whether the packet size slider should appear in the footer of the send widget
+.field
+  = f.label :default_packet_size_limit, 'Default packet size'
+  %p The initial maximum packet size.  If the control is not shown, this will be the fixed packet size for the level.  Should be a positive number, or "Infinity"
+  = f.text_field :default_packet_size_limit, placeholder: 'Default packet size'
```

Comment:
What happens if I enter a string that's not a number? I suppose we probably don't care too much since we're the consumers (and thus don't need to error proof quite so much).